tasks.py

```diff
@@ -733,3 +738,30 @@ def link_checker(ctx: Context, skip_external: bool = True):
         skip_external=skip_external,
     )
     raise invoke.Exit(message, code)
+
+
+@invoke.task(
+    aliases=("automerge",),
+)
+def show_automerges(ctx: Context):
+    """Show github pull requests currently in automerge state."""
+    url = "https://api.github.com/repos/great-expectations/great_expectations/pulls"
+    response = requests.get(
+        url,
+        params={
+            "state": "open",
+            "sort": "updated",
+            "direction": "desc",
+            "per_page": 50,
+        },
+    )
+    LOGGER.debug(f"{response.request.method} {response.request.url} - {response}")
+    pr_details = response.json()
+    LOGGER.debug(pf(pr_details, depth=2))
+
+    if automerge_prs := tuple(x for x in pr_details if x["auto_merge"]):
```

Comment:
It seemed like just checking for the presence of this key could be enough but @cdkini you probably have more experience here.